public/app/core/components/Select/FolderPicker.tsx

```diff
@@ -183,6 +196,7 @@ export class FolderPicker extends PureComponent<Props, State> {
       <div data-testid={selectors.components.FolderPicker.containerV2}>
         <AsyncSelect
           inputId={inputId}
+          isOpen={true}
```

Comment:
Any particular reason why we had to add this prop here?

Reply:
I was testing something and forgot to remove :) Tests are failing due to this